README.md

```diff
@@ -1 +1,127 @@
+},{
+  title: 'Brush teeth',
+  complete: true
+}];
+
+var list = document.querySelector('ul.todos');
+
+todos.forEach(function (todo) {
+  var item = document.createElement('li');
+  
+  var wording = todo.complete ? 'Completed!' : 'Not completed :(';
+  
+  item.innerHTML = '<h4>' + todo.title + '</h4> ' + wording;
+  
+  list.appendChild(item);
+});
+```
+
+This worked fine but what if our `todos` array was to get updated? We'd have to make sure to only insert the new items into our list, and only remove the correct elements when an item was removed from our array. This could get extremely messy and out of sync with our data if we were not really careful.
+
+## Angular saves the day!
+
+Try and contain your excitement when you see how this is done in Angular.
+
+The HTML:
+
+```html
+<ul>
+  <li ng-repeat="todo in vm.todos">
+    <h4>{{ todo.title }}</h4>
+    
+    {{ todo.complete && 'Completed!' || 'Not completed :(' }}
+  </li>
+</ul>
+```
+
+Our controller: 
+
+```js
+function TodosController() {
+  this.todos = [{
+     title: 'Learn Angular',
+     complete: false,
+   },{
+     title: 'Create GitHub profile',
+     complete: false
+   },{
+     title: 'Brush teeth',
+     complete: true
+   }];
+}
+```
+
+That's everything that we need. Not even joking.
+
+This handles everything - whenever our `todos` array is updated, Angular will update our view to match, removing and adding only the relevant nodes.
+
+## ng-model
+
+Along with `ng-repeat`, `ng-model` is one of the most commonly used directives provided by Angular. `ng-model` works it's magic by setting an input's value to a variable set in the controller, as well as updating said variable whenever the user updates the `<input />`'s value.
+
+```html
+<input ng-model="vm.username" />
+```
+
+This will populate our `<input />` with the value of `vm.username` (which could be empty), and then when the user types in the input, updates `vm.username` to match.
+
+[Check out this JSFiddle](https://jsfiddle.net/pjrfbkku/) showing the power of `ng-model`.
+
+## ng-class
+
+Another task that's quite common in modern applications is dynamically adding and removing classes depending on different variables. For example, we might want to add a complete class on our example above if the todo is complete.
+
+We can do this using the power of `ng-class`. `ng-class` takes an object, with the keys being the class you want to dynamically add, and the value being an expression - when the expression evaluates to `true`, the class is added. When it's `false`, the class will be removed.
+
+For example:
+
+```html
+<div ng-class="{'classNameHere': true, 'otherClassName': false}">
+</div>
+```
+
+In the example above, `classNameHere` would be added to our `<div />` (as the object's value is equal to true), whereas `otherClassName` wouldn't be added as the value is equal to false.
+
+We can swap out `true` and `false` for different values, such as a variable's value. Let' do this for the todo items above.
```

Comment:
This is a great breakdown  of ng-class. You start with a simple example, then show us in the context of the example you provided above. Nice one!